CPAC/pipeline/cpac_group_runner.py

```diff
@@ -1850,6 +1851,135 @@ def run_isc(pipeline_config):
+                           os.path.basename(pipeline_dir))
+
+    working_dir = os.path.join(working_dir, 'cpac_group_analysis', 'ISC',
+                               os.path.basename(pipeline_dir))
+
+    crash_dir = os.path.join(crash_dir, 'cpac_group_analysis', 'ISC',
+                             os.path.basename(pipeline_dir))
+
+    output_df_dct = gather_outputs(
+        pipeline_dir,
+        ["functional_to_standard", "roi_timeseries"],
+        inclusion_list=None,
+        get_motion=False, get_raw_score=False, get_func=True,
+        derivatives=["functional_to_standard", "roi_timeseries"],
+        exts=['nii', 'nii.gz', 'csv']
+    )
+
+    for preproc_strat in output_df_dct.keys():
+        # go over each preprocessing strategy
+
+        derivative, _ = preproc_strat
+
+        if "voxel" not in levels and derivative == "functional_to_standard":
+            continue
+
+        if "roi" not in levels and derivative == "roi_timeseries":
+            continue
+
+        df_dct = {}
+        strat_df = output_df_dct[preproc_strat]
+
+        if len(set(strat_df["Series"])) > 1:
+            # more than one scan/series ID
+            for strat_scan in list(set(strat_df["Series"])):
+                # make a list of sub-dataframes, each one with only file paths
+                # from one scan ID each
+                df_dct[strat_scan] = strat_df[strat_df["Series"] == strat_scan]
+        else:
+            df_dct[list(set(strat_df["Series"]))[0]] = strat_df
+
+        if isc:
+            for df_scan in df_dct.keys():
+                func_paths = {
+                    p.split("_")[0]: f
+                    for p, f in
+                    zip(
+                        df_dct[df_scan].participant_id,
+                        df_dct[df_scan].Filepath
+                    )
+                }
+
+                isc_wf = create_isc(name="ISC_{0}".format(df_scan), working_dir=working_dir, crash_dir=crash_dir)
+                isc_wf.inputs.inputspec.subjects = func_paths
+                isc_wf.inputs.inputspec.permutations = permutations
+                isc_wf.inputs.inputspec.std = std_filter
+                isc_wf.inputs.inputspec.collapse_subj = False
+                isc_wf.run()
+
+        if isfc:
+            for df_scan in df_dct.keys():
+                func_paths = {
+                    p.split("_")[0]: f
+                    for p, f in
+                    zip(
+                        df_dct[df_scan].participant_id,
+                        df_dct[df_scan].Filepath
+                    )
+                }
+
+                isfc_wf = create_isfc(name="ISFC_{0}".format(df_scan), working_dir=working_dir, crash_dir=crash_dir)
+                isfc_wf.inputs.inputspec.subjects = func_paths
+                isfc_wf.inputs.inputspec.permutations = permutations
+                isfc_wf.inputs.inputspec.std = std_filter
+                isfc_wf.inputs.inputspec.collapse_subj = False
+                isfc_wf.run()
+
```

Comment:
Just want to make sure nothing in run_isc_group got changed, since it seems like it was re-added in or moved at some point.